src/beautifiers/rubocop.coffee

```diff
@@ -51,9 +63,9 @@ module.exports = class Rubocop extends Beautifier
       rubocopArguments.push("--config", tempConfig) if tempConfig?
       @debug("rubocop arguments", rubocopArguments)
 
-      @run(rubocopPath, rubocopArguments, {
```

Comment:
`rubocopPath` should be deprecated and/or removed from supported options.
See https://github.com/Glavin001/atom-beautify/blob/5f968cb9f33ebd235471aaa7ad271c8292fe190e/src/beautifiers/php-cs-fixer.coffee#L91-L93 for example.

Reply:
May also want to be backward compatible for a couple versions by falling back to `@run(rubocopPath` when `rubocopPath` is set.